packages/logging/src/servicies/logger/logger.service.ts

```diff
@@ -60,8 +60,10 @@ export class LoggerService implements Logger {
     }
 
     try {
-      // eslint-disable-next-line @typescript-eslint/no-var-requires
-      return require("pino-pretty")({ colorize: true, sync: true });
```

Comment:
in order to prevent loading server-specific libraries on the client side (browser), we should set `if (typeof window === "undefined")` in front of `require`  @ygrishajev

Reply:
yea, take a look few lines higher ;)

Reply:
return is working on runtime
at compile time, require is working csr and ssr
pino-pretty should be worked with ssr
but `pino-pretty` is always importing

so we should change this code like this
```
private getPrettyIfPresent(): PinoPretty.PrettyStream | undefined {
    if (typeof window !== "undefined" || LoggerService.config.STD_OUT_LOG_FORMAT !== "pretty") {
      return;
    } else {
      try {
        // eslint-disable-next-line @typescript-eslint/no-var-requires
        return require("pino-pretty")({ colorize: true, sync: true });
      } catch (e) {
        this.debug({ context: LoggerService.name, message: "Failed to load pino-pretty", error: e });
        /* empty */
      }
    }
  }```

Reply:
@ygrishajev